src/accessibility/describe.js

```diff
@@ -251,13 +252,13 @@ function _descriptionText(text) {
  */
 
 //creates fallback HTML structure
-function _describeFallbackHTML(dummyDOM, cnvId, text) {
+function _describeFallbackHTML(dummyDOM, cnvId) {
   //if there is no description container
   if (!dummyDOM.querySelector(`#${cnvId + descContainer}`)) {
     //if there are no accessible outputs (see textOutput() and gridOutput())
     if (!dummyDOM.querySelector(`#${cnvId}accessibleOutput`)) {
       //create description container + <p> for fallback description
-      dummyDOM.querySelector('#' + cnvId).innerHTML = `<div id="${cnvId +
+      dummyDOM.querySelector(`#${cnvId}`).innerHTML = `<div id="${cnvId +
```

Comment:
before this inner if/else, maybe (on line 258):

``let descContainer = `<div id="${cnvId + descContainer}" role="region" aria-label="Canvas Description"><p id="${cnvId + fallbackDescId}"></p></div>`; ``

then use variable in if/else (unless I missed something when I read this, it looks like the HTML is the same, just how it gets inserted changes in the condition)